bazel/swift_static_framework.bzl

```diff
@@ -168,6 +170,7 @@ def swift_static_framework(
     swift_library(
         name = archive_name,
         srcs = srcs,
+        data = resources,
```

Comment:
I don't think we need this, since we'll need to manually put this in the framework bundle.